src/Sanitizer.php

```diff
@@ -107,6 +108,52 @@ public function sanitize( string $file, string $file_name ): bool {
 		return true;
 	}
 
+	/**
+	 * Add SVG to wp_get_ext_types results when it's enabled.
+	 *
+	 * @since {VERSION}
+	 *
+	 * @param array|mixed $types Array of allowed file types.
+	 *
+	 * @return array
+	 */
+	public function include_svg( $types ): array {
```

Comment:
it is not new method, I just moved it from the end of file (I like when class has methods grouped by visibility and this public method was just below private ones)